apps/labs/posts/labs-status-update-2019-04.md

```diff
@@ -0,0 +1,188 @@
+wrote a nice blog post on the highlights in that release
+[here](http://labs.quansight.org/blog/2019/04/whats-new-in-sympy-14/).
+
+Gonzalo Pena-Castellanos is working full-time on [Spyder](https://www.spyder-ide.org/),
+with guidance from Carlos Cordoba. Together they have been working very hard to get
+the first beta of Spyder 4 ready. Some exciting new features are also in the
+works, however Gonzalo will be blogging about those soon so I won't steal his
+thunder.
+
+Ivan Ogasawara is spending some time each week on maintenance of
+[Ibis](https://docs.ibis-project.org/). If you're a Pandas or scikit-learn user
+and need to interact with SQL databases or HDFS/Spark, Ibis is worth looking into.
+
+I myself have enjoyed having a little more bandwidth for NumPy and SciPy.
+On the technical front, this allowed me to contribute to the design discussion
+about an [addition](https://mail.python.org/pipermail/numpy-discussion/2019-April/079317.html)
+to NEP 18 (the `__array_function__` override mechanism),
+do the [numpydoc](https://github.com/numpy/numpydoc) 0.9 release, deal
+with several build issues, and review a number of PRs
+(the one
+allowing to specify [BLAS and LAPACK link order](https://github.com/numpy/numpy/pull/13132)
+was particularly nice). On the organizational front, I fixed the description
+of how donations are handled on numpy.org, finalized the
+[Tidelift](https://tidelift.com/) agreement for NumPy (see the
+[announcement](https://mail.python.org/pipermail/numpy-discussion/2019-April/079370.html)
+for details), helped NumPy and SciPy get accepted for the
+[Google Season of Docs](https://developers.google.com/season-of-docs/) program,
+and did everything needed to finalize the fiscal sponsorship agreement between
+SciPy and NumFOCUS.
+
+## Jupyter and JupyterLab improvements
+
+Jupyter is a key part of the PyData ecosystem. It extends well beyond that though, so I'm
+giving it its own bucket here. At Quansight we have a number of Jupyter core developers
+and contributors. Ian Rose, Saul Shanabrook, Grant Nestor and others have been very busy
+with both maintenance tasks and adding new features to Jupyter and JupyterLab.
+
+JupyterLab is about to get support for printing (not inside the notebook, but the old-fashioned
+`Ctrl-P` variant). [This pull request](https://github.com/jupyterlab/jupyterlab/pull/5850)
+by Saul has nice screenshots showing the feature in action for whole notebooks,
+ images, the JSON viewer and the inspector.
+
+Ian worked on the third alpha release of JupyterLab 1.0, on testing and CI infrastructure,
+and other general maintenance tasks. He also improved PDF preview in JupyterLab, so it
+now [works as expected](https://github.com/jupyterlab/jupyterlab/pull/6264) in Firefox
+and Chrome (at least).
+
+Saul added support for the [nteract Data Explorer](https://github.com/nteract/nteract/tree/master/packages/data-explorer) to the JupyterLab data registry as a plugin.
```

Comment:
@rgommers this nteract link is broken. Any suggestion for a replacement?

Reply:
https://data-explorer.nteract.io/